### PR TITLE
#1705-glossary additions

### PR DESCRIPTION
**Pull request by Uma-95**
hello @kathatherine 
This is the change that I have made for #1705 
https://github.com/conda-forge/conda-forge.github.io/compare/main...Uma-95:1705-glossary-additions?expand=1
Kindly review.

### CODE REVIEW

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Powercycling for CI

File: src/misc/00_intro.rst [hunk 1]
```diff
@@ -6,6 +6,12 @@ Glossary
 
 .. glossary::
 
+Conda package
+     **C**\ onda **P**\ ackage.A conda package is a .conda file that contains system-level libraries,Python or other modules,executable programs and other components,metadata under the info/ directory and a collection of files that are installed directly into an install prefix. `Learn More <https://en.wikipedia.org/wiki/Conda_(package_manager)>`__.
```
Comment (kathatherine) [inline src/misc/00_intro.rst:10]: There's some spacing issues here. Needs a space between the period and "A" at the beginning of the first sentence, and after each comma.

File: src/misc/00_intro.rst [hunk 2]
```diff
@@ -6,6 +6,12 @@ Glossary
...
+
+  Conda channels
+     **C**\ onda **C**\hannels.Conda channels are the locations where packages are stored. They serve as the base for hosting and managing packages.Conda-Forge is a conda channels.
```
Comment (kathatherine) [inline src/misc/00_intro.rst:13]: Same here. Need a space after each period.

Also "channels" at the very end of this definition should not be plural.

**Review by kathatherine — CHANGES_REQUESTED** [review]

File: src/user/introduction.rst [hunk 3]
```diff
@@ -41,6 +41,8 @@ conda-forge is a community effort that tackles these issues:
  - By default, we build packages for macOS, Linux AMD64 and Windows AMD64.
  - Many packages are updated by multiple maintainers with an easy option to become a maintainer.
  - An active core developer team is trying to also maintain abandoned packages.
+ 
+You can refer to the glossary `here<https://conda-forge.org/docs/misc/00_intro.html#glossary>`.
```
Comment (jaimergp) [inline src/user/introduction.rst:45]:
```suggestion
```

I don't think this is the place to put this reference

Comment (jaimergp) [reply src/user/introduction.rst:45]: Oops, this will break the CI I think @jakirkham. I don't think it's a valid RST ref.

Comment (jaimergp) [reply src/user/introduction.rst:45]: It didn't break but it doesn't render nicely. See last line in https://conda-forge.org/docs/user/introduction.html#why-conda-forge

**Pull request merged**